metric/unaggregated/types.go

```diff
@@ -119,14 +123,14 @@ func (m *MetricUnion) String() string {
 	}
 }
 
-// Reset resets the metric union
+// Reset resets the metric union.
```

Comment:
Sounds good to me. This case is like the one in my comment below, i.e. it's always the caller's responsibility to return the slices to the pool.